docs/tutorials/cloud-init.md

```diff
@@ -6,18 +6,18 @@ Ubuntu WSL users can now leverage it to perform an automatic setup to get a work
 
 > See more:  [cloud-init official documentation](https://cloudinit.readthedocs.io/en/latest/index.html).
 
-The latest release of Ubuntu (Noble Numbat 24.04 LTS) comes with cloud-init already preinstalled, so you'll need that specific application to follow this tutorial. Ubuntu 24.04 LTS can be installed from [this link to the Microsoft Store](https://www.microsoft.com/store/productId/9NZ3KLHXDJP5?ocid=pdpshare). A previous version of this tutorial used Ubuntu-Preview, because that comes with the latest in-development features. You can still use it to follow the instructions below, if you prefer.
+The latest release of Ubuntu (Noble Numbat 24.04 LTS) comes with cloud-init already preinstalled, so you'll need that specific application to follow this tutorial. Ubuntu 24.04 LTS can be installed from [this link to the Microsoft Store](https://www.microsoft.com/store/productId/9NZ3KLHXDJP5?ocid=pdpshare). A previous version of this tutorial used Ubuntu Preview, because that comes with the latest in-development features. You can still use it to follow the instructions below, if you prefer.
```

Comment:
Yes -- agreed! In this case it should probably be "Ubuntu (Preview)", as it also agrees with the form used earlier in the paragraph. Thanks for this.

The naming convention for product releases versus instances is something for me to note down. Our style guide has some specific guidance on naming distributions. So the example above should be Ubuntu 24.04 LTS (Noble Numbat); however, it doesn't offer guidance on something like WSL instances of Ubuntu.

It is obviously true that the correct instance name needs to be used when running a command that contains it. I think it also provides some clarity in the text if the instances are referenced in this way, although it might warrant an explanatory note to highlight the distinction up-front.

Reply:
Remember we have a reference item about this naming convention in our docs:

https://canonical-ubuntu-wsl.readthedocs-hosted.com/en/latest/reference/distributions/#naming

Reply:
Indeed.